### PR TITLE
NAS-118141 / 22.12 / allow easy checking of sha256 checksum

### PR DESCRIPTION
**Pull request by q5sys**
Joe should be discussing this PR with Waqar.

This will allow us to automate the shasum checking in our jenkins pipline since the shasum file will now be made so that we can use `shasum -c ` to verify its correct once the push has happened.
image

### CODE REVIEW

File: scale_build/image/iso.py [hunk 1]
```diff
@@ -164,8 +164,7 @@ def make_iso_file():
 
     with open(os.path.join(RELEASE_DIR, f'TrueNAS-SCALE-{get_image_version()}.iso.sha256'), 'w') as f:
         f.write(run(
-            ['sha256sum', os.path.join(RELEASE_DIR, f'TrueNAS-SCALE-{get_image_version()}.iso')], log=False
-        ).stdout.strip().split()[0])
+            ['sha256sum', os.path.join(RELEASE_DIR, f'TrueNAS-SCALE-{get_image_version()}.iso')], log=False).stdout.replace("./tmp/release/", ""))
```
Comment (sonicaj) [inline scale_build/image/iso.py:167]: Use `.replace(f'{RELEASE_DIR}/', '').strip()` instead please

**Review by sonicaj — CHANGES_REQUESTED** [review]

File: scale_build/image/iso.py (hunk 1 above)
Comment (q5sys) [reply scale_build/image/iso.py:167]: wilco

**Review by sonicaj — APPROVED** [review]
Thanks! ( do test please before merging )

**Review by sonicaj — COMMENTED** [review]
Hmm flake8 is failing, can you confirm please if everything is in order ( indent etc ) ?

**Comment by q5sys** [comment]
ah its line length... `./scale_build/image/iso.py:167:121: E501 line too long (155 > 120 characters)`

**Review by sonicaj — APPROVED** [review]

**Comment by q5sys** [comment]
Build is good and sum file is formed correctly
image

**Pull request merged**